global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "sdk": {
+    "version": "6.0.300",
+    "rollForward": "major"
+  }
```

Comment:
This was removed in #17462 but still exists here, was that intentional?

Reply:
Was an oversight, I'll pull it into a follow-up branch I have.